README.md

```diff
@@ -9,13 +9,13 @@
-- [`CommunityToolkit.Diagnostics`](https://docs.microsoft.com/en-us/windows/communitytoolkit/diagnostics/introduction): a set of helper APIs (specifically, [`Guard`](https://docs.microsoft.com/en-us/windows/communitytoolkit/developer-tools/guard) and [`ThrowHelper`](https://docs.microsoft.com/en-us/windows/communitytoolkit/developer-tools/throwhelper)) that can be used for cleaner, more efficient and less error prone argument validation and error checking.
-- [`CommunityToolkit.HighPerformance`](https://docs.microsoft.com/en-us/windows/communitytoolkit/high-performance/introduction) a collection of helpers for working in high-performance scenarios. It includes APIs such as [pooled buffer helpers](https://docs.microsoft.com/en-us/windows/communitytoolkit/high-performance/memoryowner), a fast [string pool](https://docs.microsoft.com/en-us/windows/communitytoolkit/high-performance/stringpool) type, a 2D variant of `Memory<T>` and `Span<T>` ([`Memory2D<T>`](https://docs.microsoft.com/en-us/windows/communitytoolkit/high-performance/memory2d) and [`Span2D<T>`](https://docs.microsoft.com/en-us/windows/communitytoolkit/high-performance/span2d)) also supporting discontiguous regions, helpers for bit shift operations (such as [`BitHelper`](https://docs.microsoft.com/en-us/windows/communitytoolkit/high-performance/span2d), also used in [Paint.NET](https://www.getpaint.net/)), and more.
-- [`CommunityToolkit.Common`](https://docs.microsoft.com/en-us/dotnet/api/?term=communitytoolkit.common): a set of helper APIs shared with other CommunityToolkit libraries.
+- [`CommunityToolkit.Diagnostics`](https://docs.microsoft.com/windows/communitytoolkit/diagnostics/introduction): a set of helper APIs (specifically, [`Guard`](https://docs.microsoft.com/windows/communitytoolkit/developer-tools/guard) and [`ThrowHelper`](https://docs.microsoft.com/windows/communitytoolkit/developer-tools/throwhelper)) that can be used for cleaner, more efficient and less error-prone argument validation and error checking.
+- [`CommunityToolkit.HighPerformance`](https://docs.microsoft.com/windows/communitytoolkit/high-performance/introduction) a collection of helpers for working in high-performance scenarios. It includes APIs such as [pooled buffer helpers](https://docs.microsoft.com/windows/communitytoolkit/high-performance/memoryowner), a fast [string pool](https://docs.microsoft.com/windows/communitytoolkit/high-performance/stringpool) type, a 2D variant of `Memory<T>` and `Span<T>` ([`Memory2D<T>`](https://docs.microsoft.com/windows/communitytoolkit/high-performance/memory2d) and [`Span2D<T>`](https://docs.microsoft.com/windows/communitytoolkit/high-performance/span2d)) also supporting discontinuous regions, helpers for bit shift operations (such as [`BitHelper`](https://docs.microsoft.com/windows/communitytoolkit/high-performance/span2d), also used in [Paint.NET](https://www.getpaint.net/)), and more.
```

Comment:
Why was "discontiguous" changed? That was correct.

Reply:
The linter requested the use of `discontinuous` since, `discontiguous` was of Spanish origin and may not exist in other English vocabularies such `en-in`, `en-au`, `en-us`, `en-mu`, and so on!

Reply:
I'm fine with changing this, but let's do that in a separate PR. Right now `discontiguous` is also used in the XML docs and comments for the various `Span2D<T>` types, so we'd need to also go change them.